recipes/r-prt/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "r-prt" %}
+{% set version = "0.1.5" %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 
+   - {{ cran_mirror }}/src/contrib/prt_{{ version }}.tar.gz
+   # - {{ cran_mirror }}/src/contrib/Archive/prt/prt_{{ version }}.tar.gz
+  sha256: 9151e98604ee7d183db4b0cb8f5d83770d086844fd57cb477271fccaea397e75
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-data.table
+    - r-curl
+    - r-fst
+    - r-crayon
+    - r-pillar
+    - r-tibble
+    - r-backports
+    - r-rlang
+    - r-vctrs
+    - r-cli
+    - r-assertthat
+    - r-fansi
+  run:
+    - r-base
+    - r-data.table
+    - r-curl
+    - r-fst
+    - r-crayon
+    - r-pillar
+    - r-tibble
+    - r-backports
+    - r-rlang
+    - r-vctrs
+    - r-assertthat
+    - r-cli
+    - r-fansi
+
+test:
+  commands:
+    - $R -e "library('prt')"           # [not win]
+    - "\"%R%\" -e \"library('prt')\""  # [win]
+
+about:
+  home: https://github.com/nbenn/prt/
+  summary: "Tabular Data Backed by Partitioned 'fst' Files"
+  description: |
+    Intended for larger-than-memory tabular data, 'prt' objects provide an interface to
+    read row and/or column subsets into memory as data.table objects. Data queries,
+    constructed as 'R' expressions, are evaluated using the non-standard evaluation
+    framework provided by 'rlang' and file-backing is powered by the fast and efficient
+    'fst' package.
+  license: GPL-3.0-only
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+  doc_url: https://nbenn.github.io/prt/
+  dev_url: https://github.com/nbenn/prt/
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - mlondschien
```

Comment:
For R packages, it is common to also include the `conda-forge/r` team as maintainers.